server/plugin/api.go

```diff
@@ -1203,19 +1203,50 @@ func (p *Plugin) getMilestones(c *UserContext, w http.ResponseWriter, r *http.Re
 	p.writeJSON(w, allMilestones)
 }
 
+func getRepositoryList(c *UserContext, userName string, githubClient *github.Client, allRepos []*github.Repository, opt github.ListOptions) ([]*github.Repository, error) {
```

Comment:
Same question here: what do we gain from  passing `allRepos` instead of the previous logic where `allRepos` had the outcome of the list call appended to it? Seems cleaner than passing a different `allRepos` to the `getRepositoryList` call in every loop.